instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import PostContainer from "./PostContainer";
+import SearchBar from "../SearchBar/SearchBar";
+
+const PostsPage = props => {
+  return (
+    <>
+      <SearchBar
```

Comment:
You could have had your SearchBar in app.js instead of here because your search bar is in nav.